moped-editor/src/views/projects/projectView/ProjectNameEditable.js

```diff
@@ -65,7 +65,7 @@ const ProjectNameEditable = (props) => {
         variables: {
           projectId: props.projectId,
           projectName: projectName,
-          projectNameSecondary: secondaryName,
+          projectNameSecondary: secondaryName.length > 0 ? secondaryName : null,
```

Comment:
Made this small change so we null out `secondaryName` in the DB if it is removed instead of storing an empty string. The generated column counts on `null` to include or exclude the secondary name part of the full name, and this follows how we treat other text inputs for the most part.